flaml/autogen/agentchat/responsive_agent.py

```diff
@@ -264,7 +264,7 @@ def _append_oai_message(
 
         Args:
             message (dict or str): message to be appended to the ChatCompletion conversation.
-            role (str): role of the message, can be "assistant", "user" or "function", this will overwrite the `role` in the message only when original role is not "function".
+            role (str): role of the message, can be "assistant", "user" or "function", this will overwrite the "role" in the message only when original role is not "function".
```

Comment:
```suggestion
            role (str): role of the message. It can be "assistant", "user" or "function". It will be used to overwrite the "role" in the message only when original role is not "function".
```